Vic2ToHoI4Tests/Vic2WorldTests/World/WorldTests.cpp

```diff
@@ -605,7 +605,8 @@ TEST(Vic2World_World_WorldTests, ConquerStrategiesAreConsolidated)
 
 	ASSERT_TRUE(world->getCountries().contains("ONE"));
 	ASSERT_THAT(world->getCountries().at("ONE").getAI().getConsolidatedStrategies(),
-		 testing::UnorderedElementsAre(std::make_pair(std::string("TWO"), 182)));
+		 testing::UnorderedElementsAre(
+			  std::make_pair(std::string("TWO"), Vic2::StrategyData{.value = 182, .provinces = std::vector<int>{2}})));
```

Comment:
Looks like linux didn't like the strategy data being created inside the assertion. You might need to create it as a named variable first.